contracts/GameJam.sol

```diff
@@ -1,16 +1,23 @@
 pragma solidity 0.5.11;
+import "../node_modules/openzeppelin-solidity/contracts/access/Roles.sol"; //Import Roles to implement custom Role Based Access Control
```

Comment:
We can import contracts from openzeppelin-solidity [like this](https://github.com/PaddyMc/game-tracker/blob/master/contracts/GameTracker.sol#L9):



```solidity
import "openzeppelin-solidity/contracts/access/Roles.sol";
```